### PR TITLE
Update RepoToolset to 1.0.0-beta2-62804-01

### PR DESCRIPTION
**Pull request by tmat**
Enables source build for dotnet/sdk repo with no patches.

### CODE REVIEW

**Comment by tmat** [comment]
@nguerrera @dsplaisted PTAL
@dagood

**Comment by tmat** [comment]
test Ubuntu16.04 Debug please

**Comment by nguerrera** [comment]
LGTM

@livarcocc I think this qualifies as infrastructure and therefore not subject to ask mode. I'm nevertheless a bit concerned about the churn while 2.1.300 P2 is stabilizing. I guess I don't understand why this has to happen in release/2.1.3xx and not master.

**Comment by nguerrera** [comment]
> I don't understand why this has to happen in release/2.1.3xx and not master.

Either that or ask for delay until P2 is out the door.

**Comment by nguerrera** [comment]
I approve of the actual file changes, but I'll let @livarcocc handle whether this can go in to this branch now.

**Comment by livarcocc** [comment]
I agree with @nguerrera, it would be better to wait for the prodcon build to be done before merging this change.

**Comment by tmat** [comment]
@dseefeld @mmitche  To comment on when this needs to be merged.

**Comment by dagood** [comment]
+@dleeapho too

For practical purposes, we can apply this PR as a set of patches in source-build for preview2. We were hoping to be able to build on the prodcon-built commits without patches wherever possible, though. I don't know how that goal interacts with escrow.

**Comment by dleeapho** [comment]
This change is part of the critical source-build effort that is part of 2.1.300 P2 stabilization. It is infrastructural and though @dagood is correct we could patch it, doing so significantly cuts back on the sustainability of source-build. @leecow to comment on escrow vis-a-vis infra changes.

**Comment by mmitche** [comment]
I'll defer to @leecow, but I'm okay with this change, as we are already past sdk in the build and can freeze if necessary.

**Comment by tmat** [comment]
test Ubuntu16.04 Debug  please

**Comment by tmat** [comment]
test Ubuntu16.04 Release  please

File: build/Versions.props
Comment (weshaggard) [inline build/Versions.props]: Do we need to condition these for offline build?

Comment (tmat) [reply build/Versions.props]: No, these are overridden by the toolset when building offline. https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/DefaultVersions.props#L63

**Comment by livarcocc** [comment]
@mmitche Is this a good time to merge this?

File: global.json
Comment (dagood) [inline global.json]: To avoid a patch, this should be `2.1.300-preview2-008533` to match the preview2 SDK we use in source-build.

**Review by dagood — COMMENTED** [review]
Can we get this PR in now?

File: global.json
Comment (tmat) [reply global.json]: Isn't `release/2.1.3xx-src` used for source build currently?

**Comment by livarcocc** [comment]
Up to @mmitche to say if he feels comfortable with this being merged now.

File: global.json
Comment (dagood) [reply global.json]: Actually, soon we'll have a `global.json` updater that would replace a patch, so it's probably fine to patch/branch for this change temporarily. LGTM as is.

Comment (tmat) [reply global.json]: I'm saying that we should merge `release/2.1.3xx-src` into `release/2.1.3xx` if it is possible to do so now. Then delete `release/2.1.3xx-src`.

Comment (dagood) [reply global.json]: Isn't that basically what this PR is doing? How does that relate to the SDK version?

Comment (tmat) [reply global.json]: I guess it is, but there might have been more changes in the -src branch since this PR was created. I think it would be better to create a new PR that merges the branch.

File: Directory.Build.props
Comment (dsplaisted) [inline Directory.Build.props]: This is adding a path to a Directory.Build.props in the parent folder.  It should probably be:

```xml
<MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileDirectory)Directory.Build.props</MSBuildAllProjects>
```

**Review by dsplaisted — APPROVED** [review]

**Comment by tmat** [comment]
Superseded by https://github.com/dotnet/sdk/pull/2178

**Pull request closed without merging**
